crates/rolldown/src/stages/link_stage/mod.rs

```diff
@@ -316,7 +316,7 @@ impl<'a> LinkStage<'a> {
                     match importee_linking_info.wrap_kind {
                       WrapKind::None => {}
                       WrapKind::Cjs => {
-                        stmt_info.side_effect = true;
```

Comment:
Before,   whatever the importee has side effects or not. we would include the stmt.